manifests/generated/virt-handler.yaml.in

```diff
@@ -61,11 +61,32 @@ spec:
           name: virt-share-dir
         - mountPath: /var/lib/kubevirt
           name: virt-lib-dir
+        - mountPath: /var/lib/kubevirt-node-labeller
+          name: node-labeller
+        - mountPath: /var/lib/kubevirt-node-labeller/config
+          name: node-labeller-configmap
         - mountPath: /var/run/kubevirt-private
           name: virt-private-dir
         - mountPath: /var/lib/kubelet/device-plugins
           name: device-plugin
       hostPID: true
+      initContainers:
+      - args:
+        - if [ ! -e /dev/kvm ]; then  mknod /dev/kvm c 10 $(grep '\<kvm\>' /proc/misc
```

Comment:
what would you think about making an alternate entry point script that we place in the virt-launcher pod that performs this rather than encoding it directly into the virt-handler yaml?

The the virt-handler yaml would just be calling the virt-launcher pod as a init container with a different entry-point than the one that actually launches the vmi.